webpack.config.js

```diff
@@ -105,21 +105,26 @@ const copyPluginIndex = defaultConfig.plugins.findIndex(
 );
 
 if ( copyPluginIndex > -1 ) {
-	const blockJsonPatternIndex = defaultConfig.plugins[
-		copyPluginIndex
-	].patterns.findIndex( ( pattern ) => pattern.from === '**/block.json' );
-
-	if ( blockJsonPatternIndex > -1 ) {
-		defaultConfig.plugins[ copyPluginIndex ].patterns[
-			blockJsonPatternIndex
-		] = {
-			...defaultConfig.plugins[ copyPluginIndex ].patterns[
-				blockJsonPatternIndex
-			],
-			context: resolve( pkg.config.coreThemeDir, 'blocks/' ),
-			to: resolve( pkg.config.coreThemeDir, 'dist/blocks/' ),
-		};
-	}
+	defaultConfig.plugins[ copyPluginIndex ].patterns.forEach(
+		( pattern, index ) => {
+			if ( pattern.from === '**/block.json' ) {
+				defaultConfig.plugins[ copyPluginIndex ].patterns[ index ] = {
+					...defaultConfig.plugins[ copyPluginIndex ].patterns[
+						index
+					],
+					context: resolve( pkg.config.coreThemeDir, 'blocks/' ),
+					to: resolve( pkg.config.coreThemeDir, 'dist/blocks/' ),
+				};
+			} else if ( pattern.from === '**/*.php' ) {
+				defaultConfig.plugins[ copyPluginIndex ].patterns[ index ] = {
+					from: '**/*.php',
+					noErrorOnMissing: true,
+					context: resolve( pkg.config.coreThemeDir, 'blocks/tribe' ),
+					to: resolve( pkg.config.coreThemeDir, 'dist/blocks/tribe' ),
+				};
+			}
+		}
```

Comment:
I wonder if we update the render_callback template to support any php-based blocks instead of this conditional. Thoughts?

Reply:
I did try a few ways to get these blocks working with one of them being updating the functions in `Block_Registrar.php` to try to also include the `tribe/` custom blocks and this is what I ended up landing on as the best solution. I'd be open to having BE take a look to see if they can do anything though. I've put a `@todo` in the code in the `Block_Registrar.php` file for BE to look into later. 

The main issue is that regardless of what the conditional here looks like, default WP build tools copy PHP files listed under the `render` attribute to `dist/blocks/`. So we'd either need to disable this functionality entirely (if we update `Block_Registrar.php`) or keep as-is to copy the render file to `dist/blocks/tribe`. 

ACF custom blocks don't have this issue because they use `renderTemplate`, which doesn't get picked up by the default WP build code, which is why we're easily able to update the template location.